Solutions/CohesitySecurity/readme.md

```diff
@@ -0,0 +1,20 @@
+# Cohesity SIEM/SOAR Integration with Sentinel
+
+This code integrates Cohesity Helios Cloud solution with MS Sentinel
+
+### Build Instructions
+1. Follow Tools/Create-Azure-Sentinel-Solution/V2/README.md:7 to setup build prerequisites
+2. Edit cohesity.config to replace these values with your owm
+* your_email_for_playbook@your_domain.com
+* your_support_email@your_domain.com
+* 11111111-2222-3333-4444-555555555555
+3. Run build.sh from anywhere
```

Comment:
@ishitagupta25, it should be build.ps1. Please also explain the meaning of Solutions/CohesitySecurity/build_one_solution.ps1

Reply:
build_one_solution.ps1
this script is used to build one specify solution.
the one provided by MS is to loop through all of the Solutions, and build one by one